### ISSUE TITLE
PHP 7.4 Deprecated warning

### ISSUE THREAD

**Issue by ruudboon** [phalcon/migrations#28]
When runnning the migrations under 7.4 I get
```
PHP Deprecated:  Unparenthesized `a ? b : c ? d : e` is deprecated. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /vendor/phalcon/migrations/src/Migrations.php on line 538
```
Looks like the ternary operator need to be upgraded. https://www.php.net/manual/en/migration74.deprecated.php

**Comment by Jeckerson** [comment]
Related - #32

**Comment by Jeckerson** [comment]
Resolved in #33
